src/main/kotlin/com/chillibits/coronaaid/shared/ModelToDtoMappers.kt

```diff
@@ -4,12 +4,17 @@ import com.chillibits.coronaaid.model.db.*
 import com.chillibits.coronaaid.model.dto.*
 import java.time.Instant
 import java.time.LocalDate
+import java.time.ZoneId
 import java.time.temporal.ChronoUnit
 
 fun LocalDate.yearsBetween(other : LocalDate): Long =
         ChronoUnit.YEARS.between(this, other)
 fun Instant.truncateToMidnight(): Long =
         this.truncatedTo(ChronoUnit.DAYS).toEpochMilli()
+fun Instant.truncateToLocalMidnight(): Long =
```

Comment:
Could we refactor all methods without any context to model mapping into a seperate extensions file?

Reply:
Yeah we could move these methods to a seperate file. But in the end the model mapper is the only file where these methods are used. (except the InfectedCompressed DTO)

Reply:
Would move them for better readablility and code separation. As they are extension methods, we would have no disadvantaged overhead due to the extraction.

Reply:
Thanks for resolving.